src/components/namelist/nameList.js

```diff
@@ -0,0 +1,76 @@
+import { useState, useEffect } from "react";
+import { get } from "../../service/apiClient";
+import "./nameList.css";
+import ProfileCircle from "../profileCircle";
+
+function NameList(inputText) {
+  const [users, setUsers] = useState([]);
+
+  useEffect(() => {
+    const getUserInfo = async () => {
+      const res = await get(`users`);
+      setUsers(res.data.users);
+    };
+    getUserInfo();
+  }, []);
+
+  //    {
+  //     users.map((user) => {
+  //       const userInitials =
+  //         user.firstName.match(/\b(\w)/g) + user.lastName.match(/\b(\w)/g);
+
+  //       return (
+  //         <section className="create-post-user-details student-view">
+  //           <ProfileCircle initials={userInitials} />
+  //           <p>
+  //             {user.firstName}
+  //             {user.lastName}
+  //           </p>
+  //           <div className="edit-icon">
+  //             <p>...</p>
+  //           </div>
+  //         </section>
+  //       );
+  //     });
+  //   }
+  // }
+
+  //
```

Comment:
Let's not leave dead code in main, chop this commented stuff out. We can always get it back from the git history if needed :)